pkg_config.10.xml

```diff
@@ -1508,5 +1508,29 @@
 		<required_version>2.2</required_version>
 		<configurationfile>ftpproxy.xml</configurationfile>
 	</package>
+	<package>
+		<name>dshield-sensor</name>
+		<pkginfolink></pkginfolink>
+		<descr><![CDATA[Set of scripts which automatically uploads firewall logs to DShield.<br/>
+			This package periodically processes the filter.log and submits the results in an
+			email to dshield.org.  This information is then used to track attacks and identify
+			compromised hosts.<br/>
+			]]></descr>
+		<website/>
+		<category>Network Management</category>
+		<version>2015-04-03 pkg 0.9.0</version>
+		<status>Beta</status>
+		<required_version>2.2</required_version>
+		<config_file>https://packages.pfsense.org/packages/config/dshield-sensor/dshield-sensor.xml</config_file>
+		<maintainer>robertn@the-nelsons.org</maintainer>
+		<configurationfile>dshield-sensor.xml</configurationfile>
+		<after_install_info>Please configure the SMTP E-Mail settings in System: Advanced: Notifications.  Then configure Services: DShield Sensor.</after_install_info>
+		<build_pbi>
+			<ports_before>lang/perl5.18</ports_before>
+			<port>net-mgmt/dshield-sensor</port>
```

Comment:
This port is specific to pfSense so I didn't submit it to FreeBSD.  That's why I submitted Bug #4581 to add it to pfsense-tools/pfPorts and put the comment at the top of this PR that the port has to be added first.